extensions/vscode/src/utils/get-using-dependency.ts

```diff
@@ -0,0 +1,19 @@
+import { hasDependency } from "./has-dependency";
+import { Dependency } from "../consts/dependency_enum";
+
+const equaltable = "equatable";
+const freezed = "freezed";
+const freezed_annotation = "freezed_annotation";
+
+export async function getUsingDependency() {
+  if (await hasDependency(equaltable)) {
+    return Dependency.Equatable;
+  } else if (
+    (await hasDependency(freezed)) ||
+    (await hasDependency(freezed_annotation))
+  ) {
+    return Dependency.Freezed;
```

Comment:
I would put `freezed` above `equatable`.
Have to discuss that though, but I think that if you have freezed and equatable, you are probably using `freezed` for your bloc and `equatable` for DTOs or something like that?